routes/index.js

```diff
@@ -25,10 +23,30 @@ router.get('/', function(req, res, next) {
   };
 
   /**
-   * Load the entries file
+   * Load the entries file from the database
    */
   mongoose.connect(mongoConnection);
     Entry.find({}, renderEntries);
 });
 
+//Raring Route -- Khaled
+router.post('/rating', function(req, res, next) {
+  const id = req.body.id;
+  const helpful = req.body.helpful  
+  const renderEntries = function(error, entry) {
+    if (error) {
+      console.log(error);
+      res.sendStatus(400)
+      return
+    }
+    res.send(entry)
+  };
+  mongoose.connect(mongoConnection);
+    Entry.findOneAndUpdate(
+      {_id: id}, 
+      {$inc: { [helpful ? 'helpful' : 'unhelpful' ]:  1}},
+      {new: true}
+    , renderEntries);
+});
```

Comment:
Looks like you might have some indentation issues here. Try formatting the document.